### PR TITLE
Airgap Doc Improvements

### PR DESCRIPTION
**Pull request by fritz-astronomer**
- remove old versions (0.27) from airgap bash commands in documentation
- add better image tag fetching bash snippets (utilize the `config.yaml` to better get the list of images, add `airflow`)
- add required tag to third-party-ingress

### CODE REVIEW

File: software/install-airgapped.md [hunk 1]
```diff
@@ -39,17 +39,31 @@ Astronomer's Docker images are hosted on a public registry which isn't accessibl
 You can also set up your own registry using a dedicated registry service such as [JFrog Artifactory](https://jfrog.com/artifactory/). Regardless of which service you use, follow the product documentation to configure a private registry according to your organization's security requirements.
 
 ## Step 2: Fetch Images from Astronomer's Helm Template
+**Note**: You should set up a `config.yaml` before running the below steps. Example `config.yaml` files are in the [AWS](install-aws-standard.md), [Azure](install-azure-standard.md), or [GCP](install-gcp-standard.md) install guides
```
Comment (kwcanuck) [inline software/install-airgapped.md:42]:
```suggestion
**Note**: Astronomer recommends creating a YAML configuration file before completing this task. Sample `config.yaml` files are available here: [AWS](install-aws-standard.md), [Azure](install-azure-standard.md), [GCP](install-gcp-standard.md).
```
Question: should this information be added to the Prerequisites section?

File: software/install-airgapped.md [hunk 2]
```diff
@@ -39,17 +39,31 @@ Astronomer's Docker images are hosted on a public registry which isn't accessibl
...
 
-The images and tags which are required for your Software installation depend on the version of Astronomer you're installing. Image tags are subject to change, even within existing versions, for example to resolve critical security issues, and therefore not listed here. To gather a list of exact images and tags required for your Astronomer Helm chart version, you can template the Helm chart and fetch the rendered image tags:
+**Note**: If you utilize other features such as [third party ingress controllers](./third-party-ingress-controllers.md), you made need to synchronize other images not listed with this snippet
```
Comment (kwcanuck) [inline software/install-airgapped.md:44]:
```suggestion
**Note**: If you utilize features such as [third party ingress controllers](./third-party-ingress-controllers.md), you may need to synchronize other images not listed here.
```

File: software/install-airgapped.md [hunk 3]
```diff
@@ -39,17 +39,31 @@ Astronomer's Docker images are hosted on a public registry which isn't accessibl
...
+
+The images and tags which are required for your Software installation depend on the version of Astronomer you're installing. 
+Image tags are subject to change, even within existing versions, for example to resolve critical security issues, and therefore not listed here.
```
Comment (kwcanuck) [inline software/install-airgapped.md:47]:
```suggestion
Image tags are subject to change, even within existing versions. 
```

**Comment by fritz-astronomer** [comment]
@jwitz / @kwcanuck / @paolaperaza - good to merge?

**Review by jwitz — APPROVED** [review]
This looks great @fritz-astronomer ! I jumped into the PR to do some minor formatting changes:

- Moved details about `config.yaml` into prerequisites
- Made running the commands in Step 2 into numbered steps
- Restored some formatting to code blocks so they appear inline with their numbered steps.

If everything looks good to you, feel free to merge! Or tag me if I overstepped in my edits and we need to roll something back.

**Pull request merged**